src/main/java/com/example/ludogorieSoft/village/services/VillageImageService.java

```diff
@@ -138,14 +138,22 @@ public String encodeImageToBase64(byte[] imageBytes) {
         return Base64.getEncoder().encodeToString(imageBytes);
     }
 
-    public List<VillageImageResponse> getAllVillageImages(){
+    //public List<VillageImageResponse> getAllVillageImages(){
```

Comment:
there is commented code here, if it is not going to be used it is good to delete it